blueocean-dashboard/src/main/js/components/PipelineRunGraph.jsx

```diff
@@ -173,7 +182,7 @@ export default class PipelineRunGraph extends Component {
 PipelineRunGraph.propTypes = {
     pipelineName: string,
     branchName: string,
-    runId: string,
```

Comment:
I take the point which leads to "contract properties" passed to certain ExtensionPoints. ATM we do it on free guess and all properties are passed through anyway. 

However we do not have API style declarations of extension points and the attributes we pass through. Which makes it hard for adopters to create reliable alternatives for our default implementations. 

I say ATM the graph is the only use case that we know of that had used run.id in a certain point of time. For housekeeping I prefer to get rid of it since it is redundant now because we pass full run.

As started I see your point, but see as well the urgent need to declare ExtensionPoints as API where we can deprecate certain attributes to later drop them. ...and to document them to start with,